components/_host-profiles.scss

```diff
@@ -33,6 +44,36 @@ host profiles.
 	}
 }
 
+.social-container {
+	text-align: center;
+	padding-top: 0.5em;
+
+	.social {
+    display: inline-block;
+    padding: 0 .25em;
+    text-transform: uppercase;
+    font-size: 0.6em;
+    font-weight: 600;
+    a {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3